build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1029,6 +1029,54 @@ message ActionResult {
 
   // The details of the execution that originally produced this result.
   ExecutedActionMetadata execution_metadata = 9;
+
+  // Inlined [ConsoleOutputs][build.bazel.remote.execution.v2.ConsoleOutputs]
+  // containing a series of stdout and stderr outputs in order of write from
+  // the action. The server SHOULD NOT inline `console_outputs` unless
+  // requested by the client in the
+  // [GetActionResultRequest][build.bazel.remote.execution.v2.GetActionResultRequest]
+  // message. The server MAY omit inlining, even if requested, and MUST do so if inlining
+  // would cause the response to exceed message size limits.
+  ConsoleOutputs console_outputs = 10;
+
+  // The digest for a blob containing the
+  // [ConsoleOutputs][build.bazel.remote.execution.v2.ConsoleOutputs] for the action,
```

Comment:
The downside of this approach is that it’s memory inefficient to load logs in case an application writes a lot of them. Could we consider letting this use that Protobuf RecordIO file format?

Reply:
Hmmmm... giving it some more thought.

The schema that we use is awfully similar to what Docker already uses to store stdout and stderr output of containers in logs: https://docs.docker.com/config/containers/logging/json-file/ This is I think also the format that Kubernetes uses internally, though I’m not 100% sure.

Would there be a way (and/or desire) to converge on format here? What about using the same field names in the Protobufs, so that jsonpb is capable of exchanging these formats transparently?

For the externally (non-inlined) logs, maybe it makes sense to use exactly the same format as Docker?